.ci/containers/contributor-checker/check-contributor.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+if [ -z "$GITHUB_TOKEN" ]; then
+    echo "Did not provide GITHUB_TOKEN environment variable."
+    exit 1
+fi
+if [ $# -lt 1 ]; then
+    echo "Usage: $0 pr-number"
+    exit 1
+fi
+PR_NUMBER=$1
```

Comment:
Do you also need to check that the PR doesn't already have an assignee? Otherwise, wouldn't each new push trigger this?

Reply:
Haha, good eye.  Done!